src/tourmaline/client/webhook_methods.cr

```diff
@@ -20,6 +20,8 @@ module Tourmaline
           end
         end
 
+        server = @webhook_server.not_nil!
```

Comment:
You could avoid this by changing like 12 to:

```crystal
@webhook_server = server = HTTP::Server.new do |context|`
```

Reply:
Fixed in latest commit